docker-compose.yml

```diff
@@ -1,15 +1,16 @@
 # To use:
 # Run "docker-compose build" to rebuild the app container.
-# Run "docker-compose run --rm php80" to run the test script with xdebug.
-# Run "docker-compose run --rm php81" to run the test script on 8.1.
+# Run "docker-compose run -u $(id -u ${USER}):$(id -g ${USER}) --rm php81 composer install" to install dependencies.
+# Run "docker-compose run  -u $(id -u ${USER}):$(id -g ${USER})--rm php81 vendor/bin/phpunit" to run the test script on 8.1.
```

Comment:
I've read that there are ways to move that to a config file so it doesn't have to get re-entered each time you run the container.  Do you have a preferred one?  I'd much rather bake that in than make it a necessary extra part of an already long command.  (Though I suppose I could always through a Makefile at it.)

Reply:
I usually also use a Makefile, but i wanted to change as little as possible in this case. If you want i can push one also.
Also i did not update the composer.json, should i do that here or should i do a different ticket/PR?

Reply:
Makefile it is then.  Good to know I'm not the only one that still uses old reliable tools. :-)